docs/bare-metal/README.md

```diff
@@ -0,0 +1,298 @@
+# Bare-Metal Mantl
+
+By bare-metal we mean a set of physical computers that
+have centos 7 installed on them.
+
+If you are using open-stack, Vmware, or a cloud provider, Mantl.io has terraform scripts for
+you. From a Mantl.io perspective, this doc is about setting up the inventory file
+by hand and preparing the machines to a state similar to what terraform would have
+done.
+
+These scripts were created for set of 8 Intel NUCs. Each had 16GB of RAM
+2 hyper threaded cores (I5s and I3s).  3 of them had 500 GB SSDs and 3
+have 2 TB HDs.  Your hardware can vary.  Theses systems are 4 times larger
+that the AWS systems defined in `terraform/aws.sample.tf` (m3.medium
+instances have 1 core and 4GB RAM).  In other words, your machines could
+be even smaller than our test system.   In fact, maybe you
+have a couple of physical boxes, and then created a few virtual machine vms for the controls
+on a couple of laptops. They have also been used on much larger machines.
```

Comment:
I'm not sure much information on hardware is necessary here, other than to state some reasonable minimum requirements (like what `m3.medium` specifies).